packages/cli/default/default-dev.yml

```diff
@@ -363,10 +363,12 @@ credentialIssuerLD:
     - suites:
         # - $require: '@veramo/credential-ld#VeramoEd25519Signature2018'
         # - $require: '@veramo/credential-ld#VeramoEd25519Signature2020'
-        - $require: '@veramo/credential-ld#VeramoJsonWebSignature2020'
+        # - $require: '@veramo/credential-ld#VeramoJsonWebSignature2020'
         # - $require: '@veramo/credential-ld#VeramoEcdsaSecp256k1RecoverySignature2020'
+        - $require: '@vckit/credential-data-integrity#VCkitEddsaRdfc2022'
```

Comment:
The same question above

Reply:
Yes, it can be backward-compatible with both V1 and V2 VCDM.